### ISSUE TITLE
Right-click context item to copy password

### ISSUE THREAD

**Issue by perry-mitchell** [buttercup/buttercup#245]
For accessibility purposes, the password should be copy-able from the context menu.

**Comment by perry-mitchell** [comment]
Refer to #244 for implementation notes.
